coordinator/src/node.rs

```diff
@@ -541,6 +646,233 @@ impl Node {
             self.finalize_rollover(&r.channel_id)?;
         }
 
+        if let Message::SubChannel(SubChannelMessage::CloseFinalize(_msg)) = &msg {
```

Comment:
nit: this file is already growing quite big. It would be nice if we could combine the resize logic e.g. in a resize.rs.